apps/web/services/feed/index.ts

```diff
@@ -66,24 +71,39 @@ const fetch = memoize(
 );
 
 export default class FeedService {
-  static async get({ skip, take }: { skip: number; take: number }) {
-    const threads = await fetch({ skip, take });
-
+  static async get({ lastReplyAt }: { lastReplyAt?: number }) {
     let accounts: accountsType[] = [];
     let ids: string[] = [];
 
-    threads.forEach((thread) => {
-      const { account } = thread.channel;
-      if (account && !ids.includes(account.id)) {
-        ids.push(account.id);
-        accounts.push(account);
-      }
-    });
+    const queryResult = await fetch({ lastReplyAt });
+
+    const threads = queryResult
```

Comment:
I don't think we can filter in memory, all threads could be from bots and in that case you'd get no results